dso-l1/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -428,14 +416,8 @@ private void openChannel() throws InterruptedException {
           DSO_LOGGER.debug("Trying to open channel....");
           final char[] pw;
           final String username;
-          if (config.getSecurityInfo().hasCredentials()) {
-            Assert.assertNotNull(securityManager);
-            username = config.getSecurityInfo().getUsername();
-            pw = securityManager.getPasswordForTC(config.getSecurityInfo().getUsername(), hostname, port);
-          } else {
-            pw = null;
-            username = null;
-          }
+          pw = null;
+          username = null;
           this.channel.open(infos, username, pw);
```

Comment:
can't we use this.channel.open(infos) here?

Reply:
indeed, fixing